powerline/renderer.py

```diff
@@ -458,7 +458,7 @@ def _render_length(self, theme, segments, divider_widths):
 					segment is first_segment
 					if side == 'left' else
 					segment is last_segment
-				))
+				)) * int(self.theme_config.get('outer_padding', 1))
```

Comment:
This is not the right place for defaults (especially given that you have to repeat default value twice). Mostly defaults are placed in `powerline/__init__.vim`, though it looks like theme_config currently does not allow omitting most things and rare defaults are not there. Also note that `theme_config` attribute is not used anywhere, but for defining `Theme` instances: you should take necessary value from theme, first saving it there with something like

```patch
diff --git a/powerline/theme.py b/powerline/theme.py
index 26b67a3..91ea3a2 100644
--- a/powerline/theme.py
+++ b/powerline/theme.py
@@ -69,6 +69,7 @@ class Theme(object):
 			self.cursor_space_multiplier = None
 		self.cursor_columns = theme_config.get('cursor_columns')
 		self.spaces = theme_config['spaces']
+		self.outer_padding = theme_config.get('outer_padding', 1)
 		self.segments = []
 		self.EMPTY_SEGMENT = {
 			'contents': None,
```